inclusive-design/idrc
# GH-166: Integrate UI Options (resolves #166)

## Description

Integrates UI Options into the IDRC site

## Steps to test

1. Open the UIO (Show Preferences) tab
2. Change the preference and ensure that the site is updated to match the specified preferences
3. Repeat for each page.

## Additional information

Pay careful attention to the colours applied by the contrast themes for all of the pages.

## Related issues

Resolves #166
